v03_pipeline/lib/misc/io.py

```diff
@@ -19,6 +23,29 @@
 FEMALE = 'Female'
 
 
+def validated_hl_function(
```

Comment:
Idea here is to wrap function with a call to "checkpoint", forcing evaluation.  If one of the assigned regexes matches, we throw a validation error with the paired message.

Reply:
Additionally, I don't think any of the exceptions added in this PR can contribute to our expected "support for multiple validation errors"...  any of these generally preclude further hail operations.